tests/test_if_cuda_multi_devices/test_dir/out.txt

```diff
@@ -1,16 +0,0 @@
-dpdispatcher_unittest_7
-dpdispatcher_unittest_5
-dpdispatcher_unittest_10
-dpdispatcher_unittest_12
-dpdispatcher_unittest_14
-dpdispatcher_unittest_8
-dpdispatcher_unittest_9
-dpdispatcher_unittest_6
-dpdispatcher_unittest_1
-dpdispatcher_unittest_2
-dpdispatcher_unittest_13
-dpdispatcher_unittest_15
-dpdispatcher_unittest_4
-dpdispatcher_unittest_0
-dpdispatcher_unittest_11
-dpdispatcher_unittest_3
```

Comment:
this file should not be removed

Reply:
This file would be generated every time the test is run. And it will change every time. Keeping it in git will make it be committed every time. From the error of the test it is not this file that is missing, but the one in tmp. Is there any other reason it should be kept?

Reply:
My mistake. But I can't get why it failed.

Reply:
Neither do I. I suspect there's some hidden bug in shell machine. Running the test in local machine would pass but result in some system resource warning saying some io buffer not closed.